otelcol/unmarshaler.go

```diff
@@ -45,7 +47,9 @@ func unmarshal(v *confmap.Conf, factories Factories) (*configSettings, error) {
 					Development: false,
 					Encoding:    "console",
 					Sampling: &telemetry.LogsSamplingConfig{
-						Initial:    100,
+						Enabled:    true,
+						Tick:       1 * time.Second,
```

Comment:
I think let's increase the tick and make it 10 seconds. It'll be more aligned with the current configuration on the exporter side. Otherwise, after this change, it will give us 100x more logs.